Thesis.tex

```diff
@@ -169,6 +169,14 @@ \subsection{Closed Source Software}
 \section{Mentality concerning IP rights}
 \label{sec:IPMent}
 
+As mentioned in the section \ref{sec:Patents} the use of patents and copyright were to 'promote the Progress of science and useful arts'. This was adapted from the british system forged in 1710; under this system only books, maps, and charts were protected by copyright. \parencite[3]{Cummings2010}
+
+These limitations were also applied to works produced in America. Interestingly enough, even works which were considered protected later on, were not eligible for copyright protection, if they were found to be of illicit nature.  Exactly this happened to the work \textit{Three Weeks} by Elinor Glynn, which was deemed so \textquotedblleft grossly immoral that copyright protection cannot exist\textquotedblright\ as is described in \cite{Fox1945}.
+We can conclude that piracy was if not necessarily as widespread as it is today, it was prevalent enough, that voices for judicial protection never ceased to exist.
```

Comment:
We can conclude that piracy was, if not necessarily as widespread as it is today, prevalent enough that voices for judicial protection never ceased to exist.